estimation.go

```diff
@@ -27,7 +27,7 @@ func EstimateTerraformPlan(ctx context.Context, backend Backend, plan io.Reader,
 	}
 
 	priorQueries, err := tfplan.ExtractPriorQueries()
-	if err != nil {
+	if err != nil && err != terraform.ErrNoQueries {
```

Comment:
This is not needed no?

Reply:
No, AFAIK this is needed in case there are no prior queries.

Reply:
But the `ExtractPriorQueries` doeds not return this error anymore, this was useful before but now it's not.

Reply:
Indeed, done